.github/workflows/tests.yml

```diff
@@ -20,11 +20,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        bazel_version: [6.5.0, 7.1.0]
+        bazel_version: [7.1.0]
         xcode_version: [15.2]
-        virtual_frameworks: [true, false]
-        sandbox: [true, false]
-        latest_rules: [true, false]
+        virtual_frameworks: [false]
+        sandbox: [false]
+        latest_rules: [false]
```

Comment:
Could we revert these, think we still want to test this matrix but on Bazel 7

Reply:
done